modules/cpmso-yaml-failure-domain-vsphere.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-vsphere_{context}"]
+= Sample VMware vSphere failure domain configuration
+
+On VMware vSphere infrastructure, the infrastructure resource specification, `infrastructures.config.openshift.io`, defines failure domains for your cluster. The `providerSpec` in the `ControlPlaneMachineSet` custom resource (CR) specifies names for failure domains. A failure domain is an infrastructure resource that comprises a control plane machine set, a vCenter datacenter, vCenter datastore, and a network. 
```

Comment:
The only place I see `infrastructures.config.openshift.io` with a name, it is called the [cluster-wide infrastructure CRD](https://docs.openshift.com/container-platform/4.13/post_installation_configuration/post-install-vsphere-zones-regions-configuration.html#references-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration). I don't see _infrastructure resource specification_ being used.

Reply:
I confirmed with the SME that both names are valid,but out of consistency I'll update to `[cluster-wide infrastructure CRD](https://docs.openshift.com/container-platform/4.13/post_installation_configuration/post-install-vsphere-zones-regions-configuration.html#references-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration)` after you finish your review. Thanks!

Reply:
Only instance exists in the openshift-docs repo as far as grep is concerned:
![Screenshot from 2024-01-31 17-27-59](https://github.com/openshift/openshift-docs/assets/57954076/bc1950dd-5966-48af-9843-de0e9636bf77)

Reply:
Yeah, I see it twice in that module. Where the module is titled _[Parameters for the cluster-wide infrastructure CRD](https://docs.openshift.com/container-platform/4.13/post_installation_configuration/post-install-vsphere-zones-regions-configuration.html#references-regions-zones-infrastructure-vsphere_post-install-vsphere-zones-regions-configuration)_, I thought that might be the winning entry.

Reply:
:heavy_check_mark: